### PR TITLE
Feature highway overtake scenario

### PR DESCRIPTION
**Pull request by DavidMansolino**
Linked with omichel/webots#5675

https://www.cyberbotics.com/doc/automobile/highway-overtake?version=feature-highway-overtake-scenrario

### CODE REVIEW

**Comment by DavidMansolino** [comment]
👍

File: automobile/highway-overtake.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+%figure "The Highway overtake world"
+
+![highway_overtake.png](images/highway_overtake.png)
+
+%end
+
+This world represents a simple straight highway with 4 lanes (including one emergency lane).
+A dense traffic is added to the highway thanks to the SUMO interface.
+The car will try to overtake the other vehicles using distance sensors, and a supervisor is used in order to display the radar targets seen by the car's front radar.
```
Comment (omichel) [inline automobile/highway-overtake.md]: The car &rarr; A car
I would remove the future form ("will try" &rarr; "tries").

File: automobile/highway.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## Highway
+
+%figure "The Highway world"
+
+![highway.png](images/highway.png)
+
+%end
+
+This world represents a simple highway scenario with various highway entrances and exits.
+This world showcase how it is easy to create a scenario using the predefined road segments.
```
Comment (omichel) [inline automobile/highway.md]: This world &rarr; It
&rarr; showcases
&rarr; how easy it is to

**Comment by DavidMansolino** [comment]
👍

**Pull request merged**